zh/maintain-tidb-binlog.md

```diff
@@ -1,156 +1,141 @@
 ---
-title: TiDB Binlog 运维
-summary: 了解如何在 Kubernetes 上运维 TiDB 集群的 TiDB Binlog。
+title: 部署 TiDB Binlog
```

Comment:
Shall we change the file name to `deploy-tidb-binlog` and move its position under `deploy` in `TOC`?